docs/how-tos/protocol.rst

```diff
@@ -2,3 +2,536 @@
 
 How to define a new ``Protocol``
 ================================
+
+The **gufe** :ref:`Protocols <protocol>` system is designed as an *extensible point* of the library,
+allowing developers to write their own methods for performing free energy calculations in a form that can take full advantage of the OpenFE ecosystem.
+
+Our recommendation in this how-to is to start simple;
+we will call out areas where you can iterate further on making your ``Protocol`` more sophisticated.
+
+.. image:: ../_static/gufe_protocol_diagram.svg
+    :alt: The ``gufe`` protocol system.
+
+
+Overview: What makes a ``Protocol``
+-----------------------------------
+
+A complete ``Protocol`` implementation requires several interconnected components:
+
+1. **Settings class**: A Pydantic model defining configuration parameters
+2. **Protocol class**: The main class that creates computational workflows
+3. **ProtocolUnit classes**: Individual computational steps in the workflow
+4. **ProtocolResult class**: Container for aggregated results from multiple runs
+
+The ``Protocol`` creates a :ref:`ProtocolDAG <protocoldag>` (directed acyclic graph) of :ref:`ProtocolUnit <protocolunit>` objects that define the computational workflow.
+Multiple ``ProtocolDAG`` executions can be aggregated into a single :ref:`ProtocolResult <protocolresult>` to provide statistical estimates.
+
+
+Step 1: Define your Settings
+-----------------------------
+
+First, create a settings class that inherits from :class:`.Settings`.
+This defines all the configuration parameters your protocol needs:
+
+.. code-block:: python
+
+    from gufe.settings import Settings
+    import pydantic
+    from openff.units import unit
+
+    class MyProtocolSettings(Settings):
+        # Required: number of independent simulations
+        n_repeats: int = 5
+        
+        # Protocol-specific parameters
+        simulation_length: pydantic.Field(
+            default=10.0 * unit.nanosecond,
+            description="Length of each simulation"
+        )
+        
+        equilibration_length: pydantic.Field(
+            default=1.0 * unit.nanosecond, 
+            description="Equilibration time before production"
+        )
```

Comment:
are `n_repeats`, `simulation_length`, and `equilibration length` required, or is this just an example of the type of extension one might want to do?